qiita_db/user.py

```diff
@@ -224,10 +224,21 @@ def create(cls, email, password, info=None):
         # for sql insertion
         columns = info.keys()
         values = [info[col] for col in columns]
-
+        queue = "add_user_%s" % email
+        conn_handler.create_queue(queue)
+        # crete user
         sql = ("INSERT INTO qiita.%s (%s) VALUES (%s)" %
                (cls._table, ','.join(columns), ','.join(['%s'] * len(values))))
-        conn_handler.execute(sql, values)
+        conn_handler.add_to_queue(queue, sql, values)
+        # create user default sample holder
+        sql = ("INSERT INTO qiita.analysis "
```

Comment:
I can do this, but it is also something to put in contributing.md and needs to be consistently done across the entire codebase. Again, if that's agreeable I will make the change.